scripts/wasmedge-init.sh

```diff
@@ -0,0 +1,86 @@
+  apt update && apt install git software-properties-common libboost-all-dev llvm-14-dev liblld-14-dev cmake ninja-build gcc g++ -y
+
+  REPO_CURL="https://github.com/WasmEdge/WasmEdge.git"
+  REPO_BRANCH="master"
+
+  git clone "${REPO_CURL}"
+  pushd WasmEdge
+  git checkout "${REPO_BRANCH}"
+  mkdir build && pushd build
+  cmake .. -G "Ninja" -DCMAKE_BUILD_TYPE=Release \
+    -DWASMEDGE_PLUGIN_WASI_NN_BACKEND="TensorflowLite" -DCMAKE_INSTALL_PREFIX="${WASMEDGE_PATH}"
+  ninja && ninja install
+
+  # install tflite deps
+  cp "_deps/wasmedgetensorflowdepslite-src/libtensorflowlite_c.so" "${WASMEDGE_LIB_PATH}"/
+
+  popd
+  popd
+}
+
+wasmedge_with_nn_init() {
+  curl -sLO https://github.com/WasmEdge/WasmEdge/releases/download/${WASMEDGE_VERSION}/WasmEdge-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+  curl -sLO https://github.com/WasmEdge/WasmEdge/releases/download/${WASMEDGE_VERSION}/WasmEdge-plugin-wasi_nn-tensorflowlite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+
+  tar -zxf WasmEdge-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+  tar -zxf WasmEdge-plugin-wasi_nn-tensorflowlite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+
+  mkdir -p "${WASMEDGE_BIN_PATH}"
+  mkdir -p "${WASMEDGE_LIB_PATH}"
+  mkdir -p "${WASMEDGE_PLUGIN_PATH}"
+
+  mv WasmEdge-${WASMEDGE_VERSION}-Linux/bin/* "${WASMEDGE_BIN_PATH}"/
+  if [[ -d "WasmEdge-${WASMEDGE_VERSION}-Linux/lib64/wasmedge/" ]]; then
+    mv WasmEdge-${WASMEDGE_VERSION}-Linux/lib64/wasmedge/* "${WASMEDGE_PLUGIN_PATH}"/
+    rmdir WasmEdge-${WASMEDGE_VERSION}-Linux/lib64/wasmedge # avoid mv fail when /lib/wasmedge exists
+  fi
+  mv WasmEdge-${WASMEDGE_VERSION}-Linux/lib64/* "${WASMEDGE_LIB_PATH}"/
+  mv libwasmedgePluginWasiNN.so "${WASMEDGE_PLUGIN_PATH}"/
+
+  rm -r WasmEdge-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz WasmEdge-${WASMEDGE_VERSION}-Linux/ WasmEdge-plugin-wasi_nn-tensorflowlite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+}
+
+wasmedge_tflite_deps_init() {
+  curl -sLO https://github.com/second-state/WasmEdge-tensorflow-deps/releases/download/${WASMEDGE_VERSION}/WasmEdge-tensorflow-deps-TFLite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+
+  tar -zxf WasmEdge-tensorflow-deps-TFLite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+
+  mv libtensorflowlite_c.so "${WASMEDGE_LIB_PATH}"/
+
+  rm WasmEdge-tensorflow-deps-TFLite-${WASMEDGE_VERSION}-manylinux2014_x86_64.tar.gz
+}
+
+wasmedge_lib_env_init() {
+  echo "${WASMEDGE_LIB_PATH}" >/etc/ld.so.conf.d/wasmedge.conf
+  ldconfig
+}
+
+mediapipe_custom_ops_init() {
+  url_base="https://github.com/yanghaku/mediapipe-custom-ops/releases/download"
+  tag="v0.0.1"
+  dl_filename="mediapipe_custom_ops-tf2.6.0-0.0.1-manylinux2014_x86_64.tar.gz"
+
+  curl -sLO "${url_base}/${tag}/${dl_filename}"
+  tar -zxvf "${dl_filename}"
+  rm "${dl_filename}"
+
+  mv "${WASMEDGE_WASINN_CUSTOM_OPS_LIBNAME}" "${WASMEDGE_PLUGIN_WASI_NN_TFLITE_CUSTOM_OPS_PATH}"
+}
+
+build_wasmedge_with_nn_tflite
+#wasmedge_with_nn_init
```

Comment:
The following function calls are commented out. Do we still need them?

Reply:
Now the script has two methods to install WasmEdge in Github actions:  build from source and download the release file.
Because the WasmEdge 0.12.0 and 0.12.1 will cause segment fault when running, so I use the `build-from-source` method instead of downloading the release file.  The PR https://github.com/WasmEdge/WasmEdge/pull/2360 has fixed it.
When the next version of WasmEdge has been released, I think we can delete the `build-from-source` method and use these download functions.

Reply:
The above comments should be in the file :-).

Reply:
Ok, I will add comments to it.

BTW, The scripts and other files are used to run tests in Github actions. if we publish to ```crates.io```, we only use the following files:
```toml
include = [
    "LICENSE",
    "README.md",
    "Cargo.toml",
    "/src",
]
```
You can see it in ```Cargo.toml```.